pokemongo_bot/cell_workers/pokemon_catch_worker.py

```diff
@@ -44,6 +44,7 @@
     'andor': lambda x, y, z: x and y or z
 }
 
+DEBUG_ON = False
 
```

Comment:
This is probably just a small issue, but should we introduce another debug flag here? We already have a debug flag in config now.